src/client/utils/tracking/gtm.ts

```diff
@@ -0,0 +1,58 @@
+import { TypeOfContract } from 'data/graphql'
+import { OfferData } from 'pages/OfferNew/types'
+import { isBundle, isYouth } from 'pages/OfferNew/utils'
+
+enum NoComboTypes {
+  NoCombo = 'NO_COMBO',
+  NoComboYouth = 'NO_COMBO_YOUTH',
+}
+
+type GAContractType = NoComboTypes | TypeOfContract
+
+interface GTMOfferData {
+  insurance_type: GAContractType
+  referral_code: 'yes' | 'no'
+  number_of_people: number
+  insurance_price: number
+}
+
+export interface DataLayerObject {
+  event: string
+  offerData?: GTMOfferData
+}
+
+const pushToGTMDataLayer = (obj: DataLayerObject) => {
+  window.dataLayer = window.dataLayer || []
+  window.dataLayer.push(obj)
```

Comment:
TS started to complain all of a sudden about `Property 'dataLayer' does not exist on type 'Window & typeof globalThis'` but I only moved the file into a folder? 🤷‍♂️ @palmenhq It is defined in gtm.d.ts

Reply:
i'd just cast `window` to `any` in this particular case 🙈

Reply:
yup let's just do that 😄